src/openforms/config/checks.py

```diff
@@ -52,9 +54,9 @@ def get_configuration_results(
                     "name": _("Address lookup plugins"),
                     "entries": [
                         self.get_plugin_entry(
-                            BAGCheck,
+                            BAGCheck(),
```

Comment:
I think it's better to update the protocol to define staticmethods, as that is how the actual checks are implemented. They're not really meant to be instantiated.